### ISSUE TITLE
No requirements for ESDC v2.1 

### ISSUE THREAD

**Issue by dzelge** [esa-esdl/jupyterhub-swarm#30]
- [x] Organise Meeting
- [x] Generate list of requirements

**Comment by dzelge** [comment]
# Requiremnets for ESDC v2.1

- Slightly more precise cube generation for high res cube?
- Cube loads into the ESDL viewer without skew
- EDIT: Allow variables without time coordinate
- Updated chunking and observing the xcube cube standard
- EDIT: xcube optimize teh cubes
- new varibales
- updated variables
- EDIT: Standardise names
- FAIR cubes
  - complete meta data
  - make original meta data json
  - add history
  - add info about cube generation
- EDIT: GLEAM -> update from version 3.2 to 3.3
- EDIT: ERA5 -> update time span back to 1980

**Comment by meggart** [comment]
Thanks for putting this together. I would add

* make the cube load faster by using [consolidated metadata](https://zarr.readthedocs.io/en/stable/api/convenience.html#zarr.convenience.consolidate_metadata)

Updated variables from our side would be 

* GLEAM -> update from version 3.2 to 3.3
* ERA5 -> update time span back to 1980

Don't really know what you mean with "Cube without time coordinate" Does this mean removing the time axis from some static variables? I would definitely support this idea. Do you imagine producing a separate dataset holding only the static variables or adding them to the main cube. AFAIK the xarray Dataset model allows including DataArrays sharing some, but not all axes, but I have no idea if this woukld still work with xcube.

**Comment by dzelge** [comment]
> Don't really know what you mean with "Cube without time coordinate" . 

Yes, that's what I mean:  Allow variables without time dimension. We discussed that during tehj meeting we had in Jena. Some varibales just don't need a time dimension. How we do that is a matter of discussion. 

> make the cube load faster by using consolidated metadata

This will be part of making the cubes xcube standard. xcube does allow 'non-optimized' cube how we call  them but when you create a cubeit will strongly recommend 'optimizing' the cube how we call it. Part of that optimisation is the consolidation of metadata.
